server/src/inmantals/server.py

```diff
@@ -351,12 +355,6 @@ def to_symbol_kind(value: int) -> Optional[lsp_types.SymbolKind]:
                     # the language server does not report work done progress for workspace symbol requests
                     "workDoneProgress": False,
                 },
-                "workspace": {
-                    "workspaceFolders": {
-                        "supported": True,
-                        "changeNotifications": True,
-                    }
-                },
```

Comment:
I don't get this change either.

Reply:
As far as I can tell this is only useful when a language server is responsible for multiple root folders in the workspace.
This should never be the case here, as each language server should only be responsible for only one root folder, so in a sense they don't care about changes to other folders in the workspace.
The extension on the other hand is doing all the matching of root folders to language servers and is listening to changes to the workspace folders to start and stop language servers as required.
I drew inspiration from [The multi language server example](https://github.com/Microsoft/vscode-extension-samples/tree/main/lsp-multi-server-sample)

Reply:
Ok, that's clear, thanks!